rosdep/base.yaml

```diff
@@ -3248,6 +3248,12 @@ libqglviewer-qt4-dev:
     xenial: [libqglviewer-dev-qt4]
     yakkety: [libqglviewer-dev-qt4]
     zesty: [libqglviewer-dev-qt4]
+libqglviewer2-qt5:
+  debian: [libqglviewer2-qt5]
+  ubuntu: [libqglviewer2-qt5]
+libqglviewer-qt5-dev:
```

Comment:
oops, one more, I think this should match the debian package name `libqglviewer-dev-qt5` (although the debian package name is very very strange)

Reply:
I mirrored what was done for qt4, where the same inconsistency exists. I am happy to fix it for qt5 though to mirror the debian name if desired?

Reply:
I'd mirror the debian name even if the previous key didn't.

Also, it looks like CI is unhappy with the alphabetical ordering after de707bd :(

Thanks for iterating!

Reply:
Done and thank you for the detailed feedback :-)